data/projects/create_jenkins_plugin.md

```diff
@@ -1,26 +1,45 @@
-Jenkins users should be able to see coala results right in the 
-Jenkins UI. The result of this project should be a full fledged Jenkins 
-integration, living in an own repository with a full testsuite.
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala. It also generally allows a wider range of users to use coala in their prefered way.
```

Comment:
Also what problem does this project try to fix? On conferences we encounter numerous devs that ask for a jenkins plugin but can't use coala without one.